packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -452,6 +455,22 @@ type AnalyticsDashboardView = BaseTrack & {
     };
 };
 
+type PinnedListUpdated = BaseTrack & {
+    event: 'pinned_list.updated';
+    userId: string;
+    properties: {
+        projectId: string;
+        organizationId: string;
+        source: 'homepage' | Space['name'];
+        pinnedListUuid: string;
```

Comment:
for these analytics events, we use `id` instead of `uuids` in the labels. 
```suggestion
        pinnedListId: string;
```